src/ScalarAuthClient.js

```diff
@@ -76,6 +76,35 @@ class ScalarAuthClient {
         return defer.promise;
     }
 
+    getScalarPageTitle(url) {
+        const defer = Promise.defer();
+
+        let scalarPageLookupUrl = SdkConfig.get().integrations_rest_url + '/api/widgets/title_lookup';
+        scalarPageLookupUrl = this.getStarterLink(scalarPageLookupUrl);
+        scalarPageLookupUrl += '&curl=' + encodeURIComponent(url);
+        request({
+            method: 'GET',
+            uri: scalarPageLookupUrl,
+            json: true,
+        }, (err, response, body) => {
+            if (err) {
+                defer.reject(err);
+            } else if (response.statusCode / 100 !== 2) {
```

Comment:
can 404 fall back to the widget's name, please?

Reply:
This only affects the "variable" component of the AppTile title, the widget's name will always be displayed in the AppTile menu bar. 

The variable component of the widget name is not displayed if it is the same as the "widget name". Adding it in here would just be unnecessary duplication (unless I am missing something?).

Reply:
Ah, I see what this is here for now. I thought it was replacing the existing title, but it's just a hint next to it.

Reply:
Yep :)